### ISSUE TITLE
Add Instructions about enabling *-backports and testing repo

### ISSUE THREAD

**Issue by sarim** [sarim/ibus-avro#137]
- [ ] Add note to README that ubuntu backports and debian testing repo need to be enabled.
- [ ] Detailed step by step instruction with screenshot in linux.omicronlab.com how to ubuntu backports and debian testing repo.

Also instructions about apt-pinning to users don't mess up their whole system.

**Comment by gunnarhj** [comment]
Hmm..

### backports
The reason why I didn't mention it in my PR was that the `backports` pocket is enabled by default since many Ubuntu cycles. And since `ibus-avro` doesn't exist in any other pocket in Ubuntu 16.04, 18.04 and 19.04, the lower pin-priority of `backports` doesn't matter in this case. But possibly a small note that `backports` needs to be enabled is motivated.

### testing
I understand that you think of the approach where you enable testing (or unstable) with lower pin-priority than your your main release (stable or oldstable) in order to be able to install `ibus-avro` the easy way also on stable/oldstable. Hadn't thought of that. Possibly a good idea to explain it in the instruction. A downside is that it would make the instruction a bit complex.

My assumption is that Debian users are usually more advanced than Ubuntu users, and so far I have been thinking that they usually are able to find their way to install Avro. Either by doing it 'the git way' or by simply installing the package from testing/unstable manually.

I'm ambivalent right now.

**Comment by sarim** [comment]
> The reason why I didn't mention it in my PR was that the `backports` pocket is enabled by default since many Ubuntu cycles. 

Not in kubuntu :P Need to test the other flavours of ubuntu to check what they're doing.

> I understand that you think of the approach where you enable testing (or unstable) with lower pin-priority than your your main release (stable or oldstable) in order to be able to install `ibus-avro` the easy way also on stable/oldstable.

Yes, thats how i've been installing it in buster. Also this way will be easier for people with debian based distro's, ex: Raspbian.

> My assumption is that Debian users are usually more advanced than Ubuntu users, and so far I have been thinking that they usually are able to find their way to install Avro. Either by doing it 'the git way' or by simply installing the package from testing/unstable manually.

Those who use debian as daily driver is certainly advanced users, but there are people who switches distro regularly just to "taste" :P Also Raspbian (other debian based distro users).

**Comment by gunnarhj** [comment]
> > The reason why I didn't mention it in my PR was that the `backports` pocket is enabled by default since many Ubuntu cycles.
> 
> Not in kubuntu :P

Didn't know that. :( Then it should be mentioned, of course.

> > I understand that you think of the approach where you enable testing (or unstable) with lower pin-priority than your your main release (stable or oldstable) in order to be able to install `ibus-avro` the easy way also on stable/oldstable.
> 
> Yes, thats how i've been installing it in buster. Also this way will be easier for people with debian based distro's, ex: Raspbian.

Ok, I'll stop arguing. My feeling for how Debian users handle these things is limited. Maybe it's possible to write a concise and not too complex instruction.

**Comment by sarim** [comment]
> Didn't know that. :( Then it should be mentioned, of course.

Seems like I was wrong too. It is more strange, in kubuntu live usb, backports is unchecked. But after installing the os, backports is checked. 


> Ok, I'll stop arguing. My feeling for how Debian users handle these things is limited. Maybe it's possible to write a concise and not too complex instruction.

Yeah, I plan to write a short note in this repo'r README, like
-  "You may add `testing` repo and pin it with low priority to get ibus-avro in wheezy, buster or any other debian based distro"

Detailed instructions will be on linux.omicronlab.com, I can make a more interactive UI there so user don't get flooded with many information, just when he clicks "get more info" button or something.

**Comment by gunnarhj** [comment]
But wheezy is EOL. Suppose it's "stretch" you should mention along with "buster".

**Comment by sarim** [comment]
> But wheezy is EOL. Suppose it's "stretch" you should mention along with "buster".

Yes, My mistake. I'm not that close to debian, forgot the right names :(
